src/components/Select.js

```diff
@@ -0,0 +1,183 @@
+import React from "react";
+import PropTypes from "prop-types";
+import CreatableSelect from 'react-select/creatable';
+import {components} from "react-select";
+
+const red = '#DE350B';
```

Comment:
can we define these globally? For example, we could have them in ColourStyles.js

Reply:
Also I think it would make sense to have them as chroma objects? (maybe not, I'm not sure)

Reply:
Wow smart! I think i would define them as chroma object in `ColourStyles.js` and export so that we could use them globally.